packages/datadog-plugin-cucumber/test/index.spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+
+const {
+  TEST_FRAMEWORK,
+  TEST_TYPE,
+  TEST_NAME,
+  TEST_SUITE,
+  TEST_STATUS
+} = require('../../dd-trace/src/plugins/util/test')
+const { expect } = require('chai')
+const path = require('path')
+const { PassThrough } = require('stream')
+
+const TESTS = [
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':3',
+    requireName: 'simple.js',
+    testName: 'pass scenario',
+    status: 'pass',
+    success: true
+  },
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':8',
+    requireName: 'simple.js',
+    testName: 'fail scenario',
+    status: 'fail',
+    success: false
+  },
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':13',
+    requireName: 'simple.js',
+    testName: 'skip scenario',
+    status: 'skip',
+    success: true
+  },
+  {
+    featureName: 'simple.feature',
+    featureSuffix: ':19',
+    requireName: 'simple.js',
+    testName: 'skip scenario based on tag',
+    status: 'skip',
+    success: true
+  }
+]
+
+wrapIt()
+
+describe('Plugin', () => {
+  let Cucumber
+
+  withVersions(plugin, '@cucumber/cucumber', version => {
+    afterEach(() => {
+      // > If you want to run tests multiple times, you may need to clear Node's require cache
+      // before subsequent calls in whichever manner best suits your needs.
+      TESTS.forEach((test) => {
+        delete require.cache[require.resolve(path.join(__dirname, 'features', test.requireName))]
+      })
+      return agent.close()
+    })
+    beforeEach(() => {
+      return agent.load('cucumber').then(() => {
+        Cucumber = require(`../../../versions/@cucumber/cucumber@${version}`).get()
+      })
+    })
+
+    describe('cucumber', () => {
+      TESTS.forEach(test => {
+        it(`should create a test span for ${test.featureName}${test.featureSuffix || ''}`, function (done) {
```

Comment:
do you need the fallback `|| ''` ?

Reply:
I want to make sure that `unknown` is not added in case the `featureSuffix` is skipped.